README.md

```diff
@@ -55,6 +55,7 @@ other languages while attempting to maintain the performance benefits of Actix.
 Clone the repo and cd into the repo:
 
 ```shell
+sudo apt install libsqlite3-dev
```

Comment:
Thanks for the contribution!  Could you separate this out as a sqlite-only operation?

Reply:
Updated as a  separate section